lib/ansible/module_utils/common/timeout.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2020, Matt Martz <matt@sivel.net>
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import signal
+from functools import wraps
+
+from ansible.module_utils.common.warnings import warn
+
+
+def _raise_timeout(signum, frame):
+    raise TimeoutError
```

Comment:
Apparently, Python 2 does not have a `TimeoutError`

Reply:
https://docs.python.org/3/library/exceptions.html#TimeoutError says that `TimeoutError` is an `OSError` subclass that's been added in Python 3.3

Reply:
we rand into this previously, look at the history of `facts/timeout.py`

Reply:
yea, makes sense